SOLUTIONS-InspectingACandidateList.ipynb

```diff
@@ -0,0 +1,666 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "objects_in_paper['Type']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Let's see whether we are dealing with strings or byte strings\n",
+    "objects_in_paper[0]['Type']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Byte strings it is!\n",
+    "galaxies = objects_in_paper[objects_in_paper['Type'] == b'G']\n",
+    "\n",
+    "galaxies.show_in_notebook()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Search the NAVO Registry for image resources.\n",
+    "\n",
+    "The paper selected super spirals using WISE, SDSS, and GALEX images. Search the NAVO registry for all image resources, using the 'service_type' search parameter. How many image resources are currently available?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_services = vo.regsearch(servicetype='image')\n",
+    "\n",
+    "print(f'{len(image_services)} result(s) found.')\n",
+    "\n",
+    "image_services.to_table()['ivoid', 'short_name', 'res_title']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Search the NAVO Registry for image resources that will allow you to search for AllWISE images.\n",
+    "\n",
+    "There are hundreds of image resources...too many to quickly read through. Try adding the 'keyword' search parameter to your registry search, and find the image resource you would need to search the AllWISE images."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "allwise_image_services = vo.regsearch(servicetype='image', keywords=['allwise'])#, 'NASA/IPAC Infrared Science Archive']) #, waveband='uv')\n",
+    "\n",
+    "\n",
+    "print(f'{len(allwise_image_services)} result(s) found.')\n",
+    "\n",
+    "allwise_image_services.to_table()['ivoid', 'short_name', 'res_title']\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6. Choose the AllWISE image service that you are interested in."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "allwise_image_service = allwise_image_services[0]\n",
+    "allwise_image_service.service"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 7. Choose one of the galaxies in the NED list.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.coordinates import SkyCoord\n",
+    "ra = galaxies['RA'][0]\n",
```

Comment:
The RA and DEC columns seem to really be 'RA(deg)' and 'DEC(deg)'

Reply:
I changed these from the January 2019 version, but it's what works for me in the current environment. Testable with https://mybinder.org/v2/gh/NASA-NAVO/workshop-dev/update-candidate-list-solns and it works so could it be something in your conda environment?

Reply:
It turns out that while I was using a fresh clone and environment, Astroquery had cached a response to that same query **a year ago** in ~/.astropy/cache/astroquery/Ned, so I was seeing results that were quite stale.

I'll remove the suggested code changes.

Reply:
oh my, I did not think of the astropy cache! I have been clearing mine for other reasons.

Reply:
We discussed this on our telecon and we'll make a note somewhere, probably in Markdown where we explain the NED query (QuickReference.md in the last workshop).